tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Builders/AccountControllerBuilder.cs

```diff
@@ -27,6 +28,18 @@ internal AccountControllerBuilder()
             _logoutService = Mock.Of<ILogoutService>();
             _mockPasswordService = new Mock<IPasswordService>();
             _urlHelper = Mock.Of<IUrlHelper>();
+            _disabledErrorMessage = new DisabledErrorMessageSetting()
+            {
+                EmailAddress = "Email",
+                PhoneNumber = "Phone"
+            };
+        }
+
+        internal AccountControllerBuilder WithDisabledErrorMessageSetting(string email, string phoneNumber)
```

Comment:
It might be simpler to change the parameter of this method to accept an `DisabledErrorMessageSetting` class. As the introduction of more parameters to the `DisabledErrorMessageSetting` class could potentially mean another parameter addition to this method. Whereas with a reference to the DisabledErrorMessageSetting class no change would be necessary.